src/print.js

```diff
@@ -1,2 +1,3 @@
-export default function printMe() {
+export default function print() {
```

Comment:
- [ ] Kindly, make sure to use arrow functions instead of the function keyword.

Reply:
Updated to use arraw functions.